scripts-available/CDB_SyncTable.sql

```diff
@@ -67,6 +67,19 @@ AS $$
   SELECT format('%s_%s_%s', prefix, txid_current(), (random()*1000000)::int)::NAME;
 $$ LANGUAGE sql VOLATILE PARALLEL UNSAFE;
 
+/*
+    Given a table name and an array of column names,
+    return array of column names qualified with the table name
+
+    Example of usage:
+
+       SELECT cartodb.__CDB_QualifyColumns('t', ARRAY['a','b']); --> ARRAY['t.a','t.b']
+
+*/
+CREATE OR REPLACE FUNCTION cartodb.__CDB_QualifyColumns(tablename TEXT, colnames TEXT[]) RETURNS TEXT[] AS
+$$
+    SELECT array_agg(tablename || '.' || _colname) from unnest(colnames) _colname;
```

Comment:
I think this needs a couple `quote_ident()` to escape identifiers when needed.

Reply:
Not sure, though and beware of double-quoting:

```
select quote_ident('Hola');
 quote_ident 
-------------
 "Hola"
(1 row)
```

```
select quote_ident(quote_ident('Hola'));
 quote_ident 
-------------
 """Hola"""
(1 row)
```

Also mind that `NAME` type is aimed at holding identifiers (with it's infamous 63 bytes limit).